src/parse_util.cpp

```diff
@@ -1060,6 +1060,11 @@ static bool detect_errors_in_block_redirection_list(
     return false;
 }
 
+parser_test_error_bits_t parse_util_detect_errors_ffi(const ast::ast_t* ast, const wcstring &buff_src,
```

Comment:
nit: I'm surprised this is needed, the version that passes a const reference should work fine.

Reply:
I really don't feel like fighting with autocxx over this, maybe there is a way to make it work without, I don't know autocxx/C++ well enough.

Reply:
yeah no need to bother for temporary code, it's a nit